code/game/machinery/computer/cloning.dm

```diff
@@ -322,35 +322,43 @@
 		return
 	if(isnull(subject) || (!(ishuman(subject))) || (!subject.dna))
 		if(isalien(subject))
-			set_scan_temp("Xenomorphs are not scannable.", "bad")
+			set_scan_temp("Safety interlocks engaged. Nanotrasen Directive 7b forbids the cloning of biohazardous alien species.", "bad")
 			SStgui.update_uis(src)
 			return
 		// can add more conditions for specific non-human messages here
 		else
-			set_scan_temp("Subject species is not scannable.", "bad")
+			set_scan_temp("Subject species is not clonable.", "bad")
 			SStgui.update_uis(src)
 			return
 	if(subject.get_int_organ(/obj/item/organ/internal/brain))
 		var/obj/item/organ/internal/brain/Brn = subject.get_int_organ(/obj/item/organ/internal/brain)
 		if(istype(Brn))
+			if(Brn.dna.species.name == "Machine")
+				set_scan_temp("No organic tissue detected within subject. Alternative revival methods recommended.", "bad")
+				SStgui.update_uis(src)
+				return
 			if(NO_CLONESCAN in Brn.dna.species.species_traits)
-				set_scan_temp("[Brn.dna.species.name_plural] are not scannable.", "bad")
+				set_scan_temp("[Brn.dna.species.name_plural] are not clonable. Alternative revival methods recommended.", "bad")
 				SStgui.update_uis(src)
 				return
 	if(!subject.get_int_organ(/obj/item/organ/internal/brain))
 		set_scan_temp("No brain detected in subject.", "bad")
 		SStgui.update_uis(src)
 		return
 	if(subject.suiciding)
-		set_scan_temp("Subject has committed suicide and is not scannable.", "bad")
+		set_scan_temp("Subject has committed suicide and is not clonable.", "bad")
 		SStgui.update_uis(src)
 		return
-	if((!subject.ckey) || (!subject.client))
-		set_scan_temp("Subject's brain is not responding. Further attempts after a short delay may succeed.", "bad")
+	if(HAS_TRAIT(subject, TRAIT_BADDNA) && src.scanner.scan_level < 2)
+		set_scan_temp("Insufficient level of biofluids detected within subject. Scanner upgrades may be required to improve scan capabilities.", "bad")
```

Comment:
"biofluid" doesn't sound right in the context of something stripped of its dna, i think something along the lines of.. 
`Significant DNA loss detected within the subject. Scanner upgrades may be required to improve scan capabilities`

Reply:
I discussed this with @CinnamonSnowball on Discord - Biofluids is used because the changeling uses a proboscis to suck out fluids within a person, and skeletons (Which also have BADDNA as a trait) are devoid of any fluids within them.

If you feel like it would be better to change it, I will do it in a bit. (Also would probably change that to 'DNA Corruption' instead - It doesn't make sense for a scanner upgrade to make DNA loss suddenly irrelevant.)